Tests/Functional/FunctionalSF3Test.allFeatures.expected.yaml

```diff
@@ -0,0 +1,45 @@
+operationName: 'test:everything'
```

Comment:
operationName: test:everything, i think the name can explain what is the focus of this test

Reply:
done